[11.0] [ADD] Add module Romanian Accounting Report.

feketemihai opened the pull request:
The module includes Trial Balance (from Account Group + Account Account) - to have hierarcy

dhongu commented on l10n_ro_report_trial_balance/static/src/js/l10n_ro_report_trial_balance_backend.js:
```diff
@@ -0,0 +1,108 @@
+var Widget = require('web.Widget');
+var ControlPanelMixin = require('web.ControlPanelMixin');
+var session = require('web.session');
+var ReportWidget = require('l10n_ro_report_trial_balance.l10n_ro_report_trial_balance');
```
card ca trebuie l10n_ro_report_trial_balance.l10n_ro_report_trial_balance_widget

dhongu commented on l10n_ro_report_trial_balance/report/trial_balance.py:
```diff
@@ -0,0 +1,261 @@
+            for group in groups:
+                accounts = acc_res.filtered(
+                    lambda a: a.account_id.id in
+                    group.compute_account_ids.ids)
```
compute_account_ids  nu exista in account.group

feketemihai replied:
Yep, pentru widget, compute_account_ids exista aici, https://github.com/feketemihai/account-financial-reporting/blob/e38193495b18e4e9f52112b083ae2286cc71cb98/account_financial_report/models/account_group.py#L22, si modulul depinde de account_financial_report ca sa nu adaug de 2 ori campu lrespectiv. Cel mai bine ar fii sa facem review in Engleza atat timp cat suntem putini, si o sa depinde de Maintainers..

dhongu commented on l10n_ro_report_trial_balance/wizards/trial_balance_wizard.py:
```diff
@@ -0,0 +1,101 @@
+    def button_export_html(self):
+        self.ensure_one()
+        action = self.env.ref(
+            'l10n_ro_report_trial_balance.l10n_ro_report_trial_balance')
```
action = self.env.ref( 'l10n_ro_report_trial_balance.action_l10n_ro_report_trial_balance')
Correct is action_l10n_ro_report_trial_balance

feketemihai replied:
Nu prea are importanta, daca nu e setat ia el implicit numele meodului din care face parte, daca da ia toata calea..

dhongu replied:
Eu zic ca e gresit numele actiunii.

dhongu commented on l10n_ro_report_trial_balance/report/trial_balance.py:
```diff
@@ -0,0 +1,261 @@
+                    group.compute_account_ids.ids)
+                if self.hide_account_balance_at_0:
+                    accounts = accounts.filtered(
+                        lambda a: a.balance_debit != 0 or
```
In ce modul e definit balance_debit?

dhongu replied:
a.balance_debit este de fapt a.debit_balance

feketemihai commented:
@dhongu Ai patit asa ceva cu ANAF, https://travis-ci.org/OCA/l10n-romania/jobs/353809849#L1495, la mine pe local nu am nici o eroare

Commit:
Add module l10n_ro_report_trial_balance.

Update tests data, to comply with added datas.

Add monetary widget.

Update styles.
